Install doesn't work + updates for JHipster 6 and OIDC

##### **Overview of the issue**

I tried the instructions in the README:

```
npm install -g generator-jhipster-vuejs@1.0.0-beta.2
```

This results in:

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for generator-jhipster-vuejs@1.0.0-beta.2
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mraible/.npm/_logs/2019-04-30T01_46_16_408Z-debug.log
```

##### **Motivation for or Use Case**

I was hoping to update the URL for OIDC from `/login` to `oauth2/authorization/oidc` so this blueprint is compatible with JHipster 6. This change already exists for Angular and React:

* Angular [`login.service.ts`](https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/main/webapp/app/core/login/login.service.ts.ejs#L55)
* React [`url-utils.ts`](https://github.com/jhipster/generator-jhipster/blob/a88374f1001e82177540b7d57ada6a56291f3ca3/generators/client/templates/react/src/main/webapp/app/shared/util/url-utils.ts.ejs#L24)

##### **Reproduce the error**

Try:

```
npm install -g generator-jhipster-vuejs@1.0.0-beta.2
```

--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 # JHipster-vuejs
 [![NPM version][npm-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Dependency Status][daviddm-image]][daviddm-url] [![Downloads][npmcharts-image]][npmcharts-url]
 > ## ⚠️ Status: in development
-> JHipster-vuejs, a Vue.js blueprint for JHipster. It will use [Vue.js](https://vuejs.org/) as the frontend library. Check out our latest beta release `1.0.0-beta.2` to try this out. 
+> JHipster-vuejs, a Vue.js blueprint for JHipster. It will use [Vue.js](https://vuejs.org/) as the frontend library. Check out our latest beta release `1.0.0-beta2` to try this out. 
 
 <div align="center">
   <a href="http://www.jhipster.tech/">
@@ -30,7 +30,7 @@ As this is a [JHipster](http://www.jhipster.tech/) blueprint, we expect you have
 To install this blueprint:
 
 ```bash
-npm install -g generator-jhipster-vuejs@1.0.0-beta.2
+npm install -g generator-jhipster-vuejs@1.0.0-beta2
 ```
 
 To update this blueprint:
@@ -102,7 +102,6 @@ jhipster -d --blueprint vuejs
 # License
 
 Apache-2.0 © [Deepu K Sasidharan](https://deepu.js.org)
-
 
 [npm-image]: https://img.shields.io/npm/v/generator-jhipster-vuejs/beta.svg?style=flat
 [npm-url]: https://npmjs.org/package/generator-jhipster-vuejs

